src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div 
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__imac"
+        src="./images/imac.jpeg"
+        alt="Apple computer picture"
+      >
+      <div class="card__text">
+        <p class="card__text_monoblock">
+          APPLE A1419 iMac 27" Retina <br>
+          5K Monoblock (MNED2UA/A) 
+        </p>
+        <p class="card__text_code">
+          Product code: 195434
+        </p>
+      </div>
+      <div class="stars">
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/223146202-24de5c1b-a416-4be5-9ed6-a3643a7f2db2.png)
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
```suggestion
      <img
        class="card__imac"
        src="./images/imac.jpeg"
        alt="Apple computer picture"
      >
      
      <div class="card__text">
        <p class="card__text_monoblock">
          APPLE A1419 iMac 27" Retina <br>
          5K Monoblock (MNED2UA/A) 
        </p>
        
        <p class="card__text_code">
          Product code: 195434
        </p>
      </div>
      
      <div class="stars">
```